frontend/src/App.tsx

```diff
@@ -19,13 +20,14 @@ const App = () => {
         <AuthProvider>
           <Box sx={{ marginTop: '64px' }}>
             <Switch>
-                <RestrictedRoute component={Login} exact path='/login' />
-                <PrivateRoute component={Home} exact path='/' />
-                {/* <PrivateRoute component={Home} exact path='/refer' /> */}
-                <PrivateRoute component={JobFeed} exact path='/jobs' />
-                <PrivateRoute component={Listing} exact path='/jobs/create' />
-                <PrivateRoute component={Profile} exact path='/profile' />
-                <Route component={NoMatch} /> {/* No match => page not found */}
+              <RestrictedRoute component={Login} exact path='/login' />
+              <PrivateRoute component={Home} exact path='/' />
+              {/* <PrivateRoute component={Home} exact path='/refer' /> */}
+              <PrivateRoute component={JobFeed} exact path='/jobs' />
+              <PrivateRoute component={Listing} exact path='/jobs/create' />
+              <PrivateRoute component={Profile} exact path='/profile' />
+              <PrivateRoute component={ApplicantTableView} exact path='/jobs/:id' />
```

Comment:
This path feels a little weird. It feels like it should show a job, but it shows a list of referrals. Maybe /jobs/:id/referrals?